app/javascript/src/components/Authentication/Login.jsx

```diff
@@ -1,21 +1,20 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
+import { Form, Formik } from "formik";
+import { Input as FormikInput } from "neetoui/formik";
```

Comment:
Do we need to alias the `Input` to `FormikInput` since we are not using the `Input` directly from neetoui?

Reply:
Since there is an Input in "neetoui", I thought to make it more explicit.